asset-transfer-basic/application-gateway-java/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,19 @@
+# initialize root logger with level ERROR for stdout and fout
```

Comment:
Don't think this file is essential - I'd remove this

Reply:
okay..will remove

Reply:
removed the file in bkp